python/lsst/obs/decam/instrument.py

```diff
@@ -87,3 +88,17 @@ def getRawFormatter(self, dataId):
         # local import to prevent circular dependency
         from .rawFormatter import DarkEnergyCameraRawFormatter
         return DarkEnergyCameraRawFormatter
+
+    def makeDataIdTranslatorFactory(self) -> TranslatorFactory:
+        # Docstring inherited from lsst.obs.base.Instrument.
```

Comment:
I've been ambivalent about these comments since I started seeing them in daf_butler. They seem to mostly add clutter, and provide benefit mostly for the reviewer.

Reply:
I don't love them either, but don't have an alternative that I like a lot better - the best alternative us all to start _assuming_ that if a method is undocumented on a class with a base class, the docstring is inherited, but I don't really feel our docs are in sufficiently good shape for that to be a reasonable assumption most of the time.